docs/content/test/run-tests.md

```diff
@@ -290,6 +303,26 @@ Configure Terraform to use locally-built binaries for `google` and `google-beta`
     TF_LOG=DEBUG TF_LOG_PATH=output.log TF_CLI_CONFIG_FILE="$HOME/tf-dev-override.tfrc" terraform apply
     ```
 
+### Run Tests with VCR Locally
+
+Running tests in "replaying" mode locally can sometimes be useful. In particular, it can allow you to test more quickly, cheaply, and without spinning up real infrastructure, once you've got an initial recording.
+
+It can also be helpful for debugging tests that seem to work locally, but fail in CI in replaying mode.
+   
+Make sure `$VCR_PATH` is set, and points to a local directory which exists, and set `VCR_MODE` appropriately.
+
+If you don't already have an existing cassette that's up to date, first do a run in "recording" mode:
```

Comment:
```suggestion
If you don't already have an existing cassette that's up to date, first do a run in `RECORDING` mode:
```

nit - ensure users can copy and run it directly

Reply:
thanks - I had it that way originally and thought maybe it would look too distracting, but good with it this way.